tests/e2escenarios/e2e_source_test.go

```diff
@@ -32,7 +32,7 @@ var _ = Describe("odo source e2e tests", func() {
 
 	Context("odo component creation from source", func() {
 		JustBeforeEach(func() {
-			if runtime.GOARCH == "s390x" {
+			if runtime.GOARCH == "s390x" || runtime.GOARCH == "ppc64le" {
 				Skip("Skipping test because there is no supported builder image.")
```

Comment:
I would say just change the `Skip` comment as `Skipping test specs because there is no supported builder image or component is not fully supported by odo` and add the test spec https://github.com/openshift/odo/pull/4000/files#diff-e223a048840cec18f0eb80f10c780334R88 within this context itself.

Reply:
The skip comment is existing in the code since earlier and I have just added the additional check for `ppc64le`.

Reply:
Moving the python spec from the current Context and adding that test to the Context at line number 33 would make python test to be skipped for s390x too which is not desired. Hence the changes as in original PR.